.github/workflows/ci.yml

```diff
@@ -268,11 +268,7 @@ jobs:
       - uses: rui314/setup-mold@v1
       - uses: ./.github/actions/setup-rust
       - uses: actions/checkout@v4
-      - name: License Check
-        run: cargo install --locked cargo-deny && cargo deny check
-      - uses: rustsec/audit-check@v2.0.0
```

Comment:
seems like cargo deny is the license and advisiory solution and audit check is no longer maintained